be/src/common/config.h

```diff
@@ -728,6 +728,8 @@ CONF_mInt64(experimental_s3_min_upload_part_size, "16777216");
 
 CONF_Int64(max_load_dop, "16");
 
+CONF_Bool(enable_load_colocate_mv, "false");
```

Comment:
why not use the FE configuration to enable the colocate?
I think the property should be attached to the materialized view itself.